src/lib/timing-src-connector.types.ts

```diff
@@ -0,0 +1,52 @@
+export interface PlayerControls {
```

Comment:
Should this be defined in a separate types file? It's unrelated to the `timing-src-connector`.

Reply:
I think it's related as this is the definition of the player the `TimingSrcConnector` expects so it's unique it. Unless you mean from the file-naming perspective and that it should be in `player-controls.types.ts`. I prefer less files but don't feel strongly about it either way.

Reply:
Let's just leave it as is until we have a need to use the interface outside of the `TimingSrcConnector`.